public/data/LawanuBorthakur.json

```
@@ -0,0 +1,28 @@
+{
+  "name": "Lawanu Borthakur",
+  "type": "personal",
+  "bio": "Web Developer & Open Source Enthusiast!",
+  "avatar": "https://github.com/lawanu-tech.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/lawanu-tech",
+      "icon": "github"
+    },
+    {
+      "name": "Connect me on LinkedIn",
+      "url": "https://www.linkedin.com/in/lawanu/",
+      "icon": "linkedin"
+    },
+    {
+      "name": "Follow me on Instagram",
+      "url": "https://www.instagram.com/lawanu.tech/",
+      "icon": "instagram"
+    },
+    {
+      "name": "Lawanu Tech - My Youtube Channel",
```

I think we should capitalize `t` for the `Youtube` word. :)
```suggestion
      "name": "Lawanu Tech - My YouTube Channel",
```

---

I don't think that is necessary 😅

---

I have remove the package-lock file from the pull request and updated the name to YouTube. Is it fine now?